modules/social_features/social_event/social_event.post_update.php

```diff
@@ -54,3 +54,75 @@ function social_event_post_update_10301_enable_event_enrollment(&$sandbox) {
   // Try to update the percentage but avoid division by zero.
   $sandbox['#finished'] = empty($sandbox['total']) ? 1 : ($sandbox['current'] / $sandbox['total']);
 }
+
+/**
+ * Sets all day value for events.
+ *
+ * Now we do not use the time check if it is an event for all-day, we use the
+ * 'All day' field in event, in which case we must make an entry in database
+ * (State API) for all events that were for all-day, use the old logic for
+ * checking.
+ */
+function social_event_post_update_10204_set_all_day_value(array &$sandbox): void {
```

Comment:
Maybe change this one too 🙈.

`social_event_post_update_10302_set_all_day_value`

Reply:
Done